packages/cli/src/utils/utils.ts

```diff
@@ -113,27 +114,131 @@ export function findReplace(manifest: string, replacer: RegExp, value: string):
   return manifest.replace(replacer, value);
 }
 
+export function replaceArrayValueInTsManifest(manifest: string, key: string, newValue: string): string {
```

Comment:
I think it would be better to generalise this. Im also a little confused with the naming. It says to replace an array value, but takes a key rather than an index.